libcloud.groovy

```diff
@@ -30,7 +30,11 @@ def replicate_to_clouds(pipecfg, basearch, buildID) {
     if (meta.amis) {
         replicators["☁️ 🔄:aws"] = {
             def replicated = false
-            def ids = ['aws-build-upload-config', 'aws-govcloud-image-upload-config']
+            def ids = ['aws-build-upload-config']
+            if (pipecfg.clouds?.aws?.govcloud &&
+                pipecfg[stream]?.skip_govcloud_hack != true) {
+                ids += 'aws-govcloud-image-upload-config'
+            }
```

Comment:
I'm touching this code too over in https://github.com/coreos/fedora-coreos-pipeline/pull/771

We can probably merge this and then I'll rebase that.